react-components/src/components/Reveal3DResources/Reveal3DResources.tsx

```diff
@@ -35,24 +39,28 @@ export type Reveal3DResourcesStyling = {
   groups?: FdmAssetStylingGroup[];
 };
 
-export type Reveal3DResourcesProps = {
+export type Reveal3DResourcesProps<NodeType = any> = {
```

Comment:
We can change to `NodeType = { [key: string]: any }` but can be postponed for later.

Reply:
Alright  :+1: I say we fix later when we have more feedback on the table